ansible/roles/graph-protocol/files/values-prd.yaml

```diff
@@ -1,12 +1,7 @@
 # ethereum network
-network: xdai
+ethereumNetwork: xdai
 rpcNode: xdai-nethermind.xdai
-# subgraph values
-subgraph_name: circlesubi/circles-ubi
-subgraph_network: xdai
-proxy_factory_address: "0x8b4404DE0CaECE4b966a9959f134f0eFDa636156"
-start_block: "12529458"
-hub_address: "0x29b9a7fBb8995b2423a71cC17cf9810798F6C543"
```

Comment:
Why don't we maintain these config values? They are reused in both sobgraph jobs, and the ones that change can be personalized here too

Reply:
We are adding the configuration with graph-protocol, thus we add them in the yamls